mlflow/tracking/client.py

```diff
@@ -706,6 +708,14 @@ def log_metric(
         :param timestamp: Time when this metric was calculated. Defaults to the current system time.
         :param step: Integer training step (iteration) at which was the metric calculated.
                      Defaults to 0.
```

Comment:
@sagarsumant @chenmoneygithub Can we include a `:return:` docstring explaining the return type(s)?